lib/ibm/cloud/sdk/power_iaas.rb

```diff
@@ -15,6 +15,7 @@ def initialize(region, guid, token, crn, tenant)
           @region = region
           @token  = token
           @tenant = tenant
+          RestClient.log = log
```

Comment:
Where is this `log` variable coming from?  I don't see it in the initialize options.  Did you mean `RestClient.log = logger` maybe?  That would be available if you did `include Logging` in this class.

Reply:
My bad, was done in a hurry, It should be
RestClient.log = logger
and I should include Loggi9ng.